rdl.gemspec

```diff
@@ -21,4 +21,6 @@ EOF
   s.add_runtime_dependency 'parser', '~>2.3', '>= 2.3.1.4'
   s.add_runtime_dependency 'sql-parser', '~>0.0.2'
   s.add_runtime_dependency 'method_source'
+  s.add_development_dependency 'colorize', '~>0.8', '>= 0.8.1'
+  s.add_development_dependency 'coderay', '~>1.2', '>= 1.1.2'
```

Comment:
@jeffrey-s-foster The code in `test_infer.rb` uses these to print out information about failed tests

Reply:
I'm mildly nervous about adding a new way to print type error messages that's only used in one place in the code. Is this something we should use more generally?

Reply:
Ah, so these aren't used for printing type information as much as they're used to provide descriptive output when a test fails. `colorize` makes it easy to change the color of text printed to the console (which I used to colorize the words "Given," "Expected," and "Got," in an effort to make the output easier to parse), and `coderay` prints colorized Ruby code to the console, which I use to print the actual method that was being inferred.

These were both in an effort to make it quick and easy to parse what test had failed, and hopefully give enough information to form an intuition as to what happened.

We could perhaps integrate something like this into the other tests, or we could do without them and just print plain text to the console.